notebooks/swarm.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.            agent_message = self.llm.run(messages=[self._system_message] + messages, tools=self.tools or None)["replies"][0]
</pre><p>Is this or None really necessary? We set it to None in post_init</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>right!</p>